plugins/network/commands

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
Yup, totally cool with the calls to plugin triggers. I was specifically referring to the plugin only code. e.g. commands and such